NeuroID/src/test/java/com/neuroid/tracker/NeuroIDClassUnitTests.kt

```diff
@@ -689,6 +690,18 @@ open class NeuroIDClassUnitTests {
         assertEquals(true, version?.contains("5.android-rn"))
     }
 
+    @Test
+    fun test_setVariable() {
+        NeuroID._isSDKStarted = true
+        setMockedNIDJobServiceManager(false)
+        setMockedDataStore()
+
+        NeuroID.getInstance()?.setVariable("test", "value")
+
+        assertEquals(1, storedEvents.count())
+        assertEquals(true, storedEvents.firstOrNull()?.type === SET_VARIABLE)
```

Comment:
Very minor, but maybe add a test to check the actual key/value of the set_variable?

Reply:
updated